src/font/fdue.rs

```diff
@@ -51,6 +77,32 @@ impl Font {
 
         Ok(Font { inner })
     }
+
+    fn try_find_best_font(name: &str) -> Vec<String> {
+        const COUNT_MATCHES: usize = 5;
+
+        FONTCONFIG
+            .cache
+            .list()
+            .keys()
+            .filter(|v| v.name.is_some())
+            .map(|font| {
+                let current = font.name.as_ref().unwrap().as_str();
+                if let Some(matching) = best_match(name, current) {
```

Comment:
It seems fuzzy search perform poorly for that problem. Tried different options and it seems simple thing like Levenshtein distance works great. So simply use `lev_distance::lev_distance` there and reverse the order (min distance -> better).